deploy/kubernetes/shared-settings.yaml

```diff
@@ -15,6 +15,8 @@ NUM_NODES_PER_NODE_POOL: 4    # used when creating nodes for gcloud deployments
 
 GUNICORN_WORKER_THREADS: 4    # number of webserver threads
 
+POSTGRES_USERNAME: postgres
```

Comment:
this is actually hardcoded in some of our manage commands so making it a secret is unnecessary